src/main/python/msmarco/retrieve.py

```diff
@@ -43,10 +39,10 @@
     total_start_time = time.time()
 
     searcher = pysearch.SimpleSearcher(args.index)
-    searcher.set_bm25_similarity(float(args.k1), float(args.b))
+    searcher.set_bm25(float(args.k1), float(args.b))
```

Comment:
Since we are here, add type=float to these arguments and remove the unnecessary `float`s here?